src/Form/ContentSelectForm.php

```diff
@@ -39,12 +39,14 @@ public function buildForm(array $form, FormStateInterface $form_state, array &$i
       '#options' => $install_options,
     ];
 
-    $form['settings'] = [
-      '#type' => 'checkboxes',
-      '#title' => $this->t('Choose settings to import'),
-      '#default_value' => array_keys($settings_options),
-      '#options' => $settings_options,
-    ];
+    if (\Drupal::state()->get('openy_preset') != 'standard') {
```

Comment:
Also, on the step "3rd Party Services" there is a field which is called "AddThis public id". IT should be hidden as well in case we do not install AddThis. You can make a condition by checking a module "openy_addthis" instead of openy preset